test-configs.yaml

```diff
@@ -1733,6 +1738,10 @@ test_configs:
     test_plans:
       - baseline
 
+  - device_type: imx8mq-zii-ultra-zest
```

Comment:
It appears to be failing to boot with all the staging jobs:
https://hekla.openlab.pengutronix.de/scheduler/job/392159
The health checks are looking fine though.  Can you please take a look?